diff --git a/plugins/next/index.js b/plugins/next/index.js
index 64f5c664ba5550da80767ff9134f59368813fada..23d0ade7ae73bf38f0f4d081d7f4af4b3f553f46 100644
--- a/plugins/next/index.js
+++ b/plugins/next/index.js
@@ -1,9 +1,11 @@
-const { startDevServer } = require('@cypress/webpack-dev-server')
+
 const findNextWebpackConfig = require('./findNextWebpackConfig')
 
 module.exports = (on, config) => {
   on('dev-server:start', async (options) => {
-    return startDevServer({ options, webpackConfig: await findNextWebpackConfig(config) })
+	const webpackConfig=await findNextWebpackConfig(config)
+	const { startDevServer } = require('@cypress/webpack-dev-server')
+    return startDevServer({ options, webpackConfig })
   })
 
   config.env.reactDevtools = true
